json-lint/schemas/inventory/adservers/a9.json

```diff
@@ -0,0 +1,54 @@
+{
+  "$id": "#a9",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "title": "Amazon adserver",
+  "description": "https://github.com/ampproject/amphtml/blob/master/ads/a9.md",
+  "additionalProperties": false,
+  "required": [
+    "type",
+    "data-amzn_assoc_ad_mode"
+  ],
+  "defaultSnippets": [
+    {
+      "label": "Amazon template",
+      "description": "contains required keys",
+      "body": {
+        "type": "a9",
+        "data-amzn_assoc_ad_mode": "$1"
+      }
+    }
+  ],
+  "properties": {
+    "type": {
+      "enum": [
+        "a9"
+      ]
+    },
+    "width": {
+      "$ref": "./types/dimension.json"
+    },
+    "height": {
+      "$ref": "./types/dimension.json"
+    },
+    "data-amzn_assoc_ad_mode": {
+      "enum": [
+        "search",
+        "manual",
+        "auto"
+      ]
+    },
+    "data-recomtype": {
+      "type": "string"
+    },
+    "data-adinstanceid": {
+      "type": "string"
+    },
+    "data-regionurl": {
+      "type": "string"
+    },
+    "data-divid": {
+      "type": "string"
+    }
```

Comment:
can we add the remaining keys that are valid for the amp-ad ? 
https://github.com/ampproject/amphtml/blob/master/ads/a9.md

data-amzn_assoc_asins
data-amzn_assoc_placement
data-amzn_assoc_search_bar     >> boolean
data-amzn_assoc_tracking_id
data-amzn_assoc_search_bar_position
data-amzn_assoc_ad_mode
data-amzn_assoc_ad_type
data-amzn_assoc_marketplace
data-amzn_assoc_region
data-amzn_assoc_title
data-amzn_assoc_default_search_phrase
data-amzn_assoc_default_category
data-amzn_assoc_linkid
data-regionurl